### DISCUSSION TITLE
Favicon invisible in Chrome dark mode

### DISCUSSION THREAD

**Discussion by jvalcher** [community/community#12744] [Other Feature Feedback, Questions, & Ideas] [answered]
Just a minor styling issue. Setting Chrome to dark mode makes GitHub's favicon in the tab and in the bookmark bar all but invisible. It looks fine in Firefox's dark mode. Sorry in advance if this isn't the right place to report this.

![Screenshot from 2022-03-11 20-57-04](https://user-images.githubusercontent.com/20879732/158001118-01a53c16-9664-4410-a9aa-269d8ded621e.png)
![Screenshot from 2022-03-11 20-58-02](https://user-images.githubusercontent.com/20879732/158001140-0baed259-e6bb-4010-a991-6a70996e3c5b.png)

**Comment by nazar1ua** [comment]
@github-staff It is important!

**Reply by nazar1ua** [reply]
Sorry, it is chrome's problem, not your's

**Comment by nazar1ua** [comment]
I have an answer: add this code to chrome (linux) `.desktop` file (default `/usr/share/applications/google-chrome.desktop`)

-----------------------
Before:
```desktop
Exec=/usr/bin/google-chrome-stable %U
```

After:
```desktop
Exec=/usr/bin/google-chrome-stable --enable-features=WebUIDarkMode --force-dark-mode %U
```

**Comment by MarkJeronimus** [comment]
Does not look fine anymore in Firefox 104-105 dark mode (maybe earlier). I remember it was fine in the past.

**Comment by Mehdimome** [comment]
در تاریخ سه‌شنبه ۴ اکتبر ۲۰۲۲،‏ ۱۴:۱۶ MarkJeronimus <
***@***.***> نوشت:

**Comment by Anmol-Baranwal** [comment, marked as answer]
It seems to be working fine for me. Rather than using the chrome dark mode, try using the system force dark mode. I think that will solve the issue.

**Reply by jvalcher** [reply]
I forgot I had posted this. It looks good now, although sometimes it goes dark when I click on it temporarily but then it goes back. Anyway no complaints. Thanks for the fix GitHub team!

![image](https://user-images.githubusercontent.com/20879732/193940737-87cb8f29-5561-4485-b310-c2eb56f29b76.png)

**Comment by MarkJeronimus** [comment]
It works correctly at home where I have the same OS, same browser, same themes, etc, etc. There's some unknown variable at play here.

**Comment by dotjax** [comment]
Can confirm it's not working for me.

![image](https://user-images.githubusercontent.com/12740958/233812383-938286ab-9a90-4f12-a691-53d2fc3ee66e.png)


OS: Linux Mint 21.1
CPU: 2700x
GPU: RX 580 GB
Browser: Version 112.0.5615.165 (Official Build) (64-bit)

What's weird is it DOES work on Edge.

**Comment by bartekpacia** [comment]
It's a bit broken. When you change to light/dark mode, the icon stays in the previous mode until you click on it.

**Comment by ablakey** [comment]
This is broken for me in Chrome. Here's some details

Chrome 114
Ubuntu Jammy

If Ubuntu is in Dark mode (set in the Ubuntu settings), the Favicon is the invisible one. Almost as if they expected Chrome to change the colour of the background of the Bookmark bar, but it does not do that.

If I change Ubuntu back to light mode, the favicon is black and visible again.

I don't want to set Light mode for Ubuntu as that makes a ton of the UI painfully bright.  I just want Chrome or GitHub to not do "clever" stuff.

Ubuntu Dark Mode:
![image](https://github.com/community/community/assets/9021944/b8901ea3-a46b-49ee-b4d1-aef73ca23e30)

Ubuntu Light Mode:
![image](https://github.com/community/community/assets/9021944/42dfa7ae-26f4-4739-8405-aa3e3300b0e0)

**Comment by AddisonG** [comment]
This was a weird one in Firefox. Buckle up.

Firefox has the following setting under `general`:

![image](https://github.com/user-attachments/assets/bf52bd9c-1a59-471a-b10e-261cccfa3acd)

This is how a **website** displays, **not the browser**!

However, GitHub looks at this setting, and changes its favicon based on that.

Which means that if your BROWSER is in light mode, but you've set your WEBSITES to display in dark mode, you'll get this:
![image](https://github.com/user-attachments/assets/d4d755b5-ae0f-4268-a1c7-3db2f11259f6)

Or, if you've got the opposite, you'll get this:
![image](https://github.com/user-attachments/assets/925fd929-a2a6-4746-9477-87d1cb9b4e63)

Both of which are extremely ugly and barely visible.

Thus, the solution is to make sure you set the theme of your browser to match your website display - You can do this by clicking that little blue link at the bottom of the first screenshot.

With that, you can get the correct favicon:
![image](https://github.com/user-attachments/assets/5b4b26a6-c353-450f-9a43-41ab1a7ab127)
![image](https://github.com/user-attachments/assets/b74e417a-d8eb-4b43-aa17-9650e786663d)

**Reply by tazihad** [reply]
I always have this no matter what. 

![Screenshot_20250525_123723](https://github.com/user-attachments/assets/65952cd1-1a3b-42bd-bceb-b60d099bc174)

**Comment by jarovarga** [comment]
It is now August 2026, more than four years after this issue was first reported, and GitHub’s favicon still does not work correctly in Firefox (155.0b5). It can remain barely visible depending on the browser theme and website colour-scheme settings. Requiring users to keep those settings in sync is a workaround, not a fix. For a platform of GitHub’s size, leaving such a basic UI and accessibility issue unresolved for this long is frankly embarrassing. Please investigate and fix it properly.

**Comment by jvalcher** [comment]
Completely forgot about this post. Looks fine on Chrome these days, so I'm going to go ahead and close this. Sorry for any inconvenience to the Firefox users. Keep fighting the good fight.
